nncf/experimental/torch/fx/node_utils.py

```diff
@@ -49,3 +54,34 @@ def get_tensor_constant_from_node(constant_node: torch.fx.Node, model: torch.fx.
             raise RuntimeError(f"Node referenced nonexistent target {'.'.join(target_atoms[:i])}")
         attr_itr = getattr(attr_itr, atom)
     return attr_itr
+
+
+def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+    """
+    Returns True if the node has a bias, False otherwise.
+
+    :param node: Target node.
+    :param nncf_graph: Target nncf_graph.
+    :return: True if the node has a bias, False otherwise.
+    """
+    # Assumes that all biases were unfused
+    if node.metatype in (om.PTConv1dMetatype, om.PTConv2dMetatype, om.PTConv3dMetatype, om.PTLinearMetatype):
```

Comment:
These metatypes should be placed in file like `groups.py`, and `OPERATIONS_WITH_BIAS` variable.